tests/security/selinux/selinux_smoke.pm

```diff
@@ -0,0 +1,54 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Test with SELinux is "enabled" and in "permissive" mode the
+#          system still work.
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#40361, tc#1682591
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils;
+use registration "add_suseconnect_product";
+use version_utils "is_sle";
+
+sub run {
+    select_console "root-console";
+
+    # make sure SELinux is "enabled" and in "permissive" mode
+    validate_script_output("sestatus", sub { m/SELinux\ status: .*enabled.* Current\ mode: .*permissive/sx });
+
+    # refresh & update
+    zypper_call("ref");
+    zypper_call("up");
+
+    # install & remove pkgs, e.g., apache2
+    zypper_call("in apache2");
+    zypper_call("rm apache2");
```

Comment:
Could you please explain the goal of installing and removing apache2 and relation to SELinux?

Reply:
This test case aims to verify the OS still works after enabled SELinux (but in permissive mode) and rebooted.
We just kind of randomly chose "zypper in/ rm apache2" for easy/smoking testing, installing and removing pkgs will access to files which SELinux policy restricted.

Reply:
OK, could you please more description about it into summary? Because "We just kind of randomly chose" doesn't look like proper design decision for the test.

Reply:
Got it, I will add more description. Thanks!

Reply:
Thanks for the update of description, it is better now.